packages/shared/src/referralCode/type.ts

```
@@ -182,6 +182,31 @@ export interface IHardwareSalesRecord {
   items: IHardwareSalesRecordItem[];
 }
 
+
+export interface IInvitePaidItem {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)